doc/03-Configuration.md

```diff
@@ -0,0 +1,28 @@
+# Configuration
+
+This is how you configure your Icinga DB so it works right off the bat.
+
+## Database
+
+Select the database resource for your setup.
+
+## Redis
+
+Configure the Redis connections details of your Icinga host.  
+If you are running a master-master setup, the first 2 fields are for your configuration master.
+
+
+Option             | Description
+-------------------|------------------------
+Redis Host         | IP adress or FQDN, usually this is your Icinga Master host
+Redis Port         | The port over which you can connect to the Redis
+
+## Command Transports
+
+Set command transports for specific Icinga instances.  
```

Comment:
A more detailed introduction would help, at the very moment starting fresh with Icinga I have no idea about this term.